tajo-plan/src/main/java/org/apache/tajo/plan/expr/FunctionEval.java

```diff
@@ -40,13 +40,27 @@
   @Expose protected FunctionDesc funcDesc;
 	@Expose protected EvalNode [] argEvals;
 
+  private transient Tuple params;
```

Comment:
It's transient field as it declared, which should not be included in hash or equals calculation.